src/user-status/UserStatusScreen.js

```diff
@@ -109,4 +113,4 @@ class UserStatusScreen extends PureComponent<Props, State> {
 
 export default connect(state => ({
   userStatusText: getSelfUserStatusText(state),
-}))(UserStatusScreen);
+}))(injectIntl(UserStatusScreen));
```

Comment:
We have a less intrusive convention for this. See the documentation for `type GetText` in `src/types.js`.